exercises/house/canonical-data.json

```diff
@@ -1,10 +1,7 @@
 {
   "exercise": "house",
-  "version": "2.1.0",
+  "version": "2.2.0",
   "comments": [
-    "JSON doesn't allow for multi-line strings, so all verses are presented ",
```

Comment:
Any reason to remove this comment? As far as I can tell, it still holds true when multiple verses are recited. The [food-chain](https://github.com/exercism/problem-specifications/blob/master/exercises/food-chain/canonical-data.json#L5) canonical data also has this comment.